test/features/pagecontent/redirects.js

```diff
@@ -68,6 +68,17 @@ describe('Redirects', function() {
         });
     });
 
+    it('should redirect to commons for missing file pages, dewiki', function() {
+        return preq.get({
+            uri: server.config.hostPort + '/de.wikipedia.org/v1/page/html/Datei:Name.jpg'
+        })
+        .then(function(res) {
+            assert.deepEqual(res.status, 200);
```

Comment:
Why is the return code 200 if it's a redirect and we haven't specified `redirect=false` in the request?